libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/initializeDashboardHandler/index.ts

```diff
@@ -269,6 +275,11 @@ function* initializeNewDashboard(
             }),
             uiActions.setMenuButtonItemsVisibility(config.menuButtonItemsVisibility),
             renderModeActions.setRenderMode(config.initialRenderMode),
+            dashboardPermissionsActions.setDashboardPermissions({
```

Comment:
@ivanmjartan I've added this for new dashboard. So for the author we assume he will have all the permissions for new his dashboard. Once the dashboard is saved the author should get all the permissions for real in the backend- in metadata.